patches/minecraft/net/minecraft/world/level/block/LiquidBlock.java.patch

```diff
@@ -1,10 +1,16 @@
 --- a/net/minecraft/world/level/block/LiquidBlock.java
 +++ b/net/minecraft/world/level/block/LiquidBlock.java
-@@ -35,6 +_,7 @@
+@@ -30,11 +_,12 @@
+ 
+ public class LiquidBlock extends Block implements BucketPickup {
+    public static final IntegerProperty f_54688_ = BlockStateProperties.f_61422_;
+-   protected final FlowingFluid f_54689_;
++   private final FlowingFluid f_54689_; // FORGE: Reduced visibility to support the Supplier constructor. Needs to be private for the field-to-method transformer.
+    private final List<FluidState> f_54691_;
     public static final VoxelShape f_54690_ = Block.m_49796_(0.0D, 0.0D, 0.0D, 16.0D, 8.0D, 16.0D);
     public static final ImmutableList<Direction> f_181233_ = ImmutableList.of(Direction.DOWN, Direction.SOUTH, Direction.NORTH, Direction.EAST, Direction.WEST);
  
-+   @Deprecated  // Forge: Use the constructor that takes a supplier
++   @Deprecated  // FORGE: Use the constructor that takes a supplier
```

Comment:
Is this change useful?

Reply:
I can revert it, I changed it to match the other of the Deprecation comment in the class.
It's a arbitrary change, and I can change it back if the forge team says so.